profile/README.md

```diff
@@ -0,0 +1,28 @@
+<img align="right" width="33%" style="margin-bottom: 2em" src="http://owlbert.io/images/owlberts-png/Popper.psd.png">
+
+# Hi, we're ReadMe! <img height=36 align=center src=https://cliply.co/wp-content/uploads/2019/06/391906110_WAVING_HAND_400px.gif>
+
+Welcome to our GitHub org! [At ReadMe][readme], we want developers to succeed. That's why building thoughtful developer experiences is at the core of everything we do.
+
+### And by “developer”, we mean a _lot_ of people…
+
+:memo:   [Anyone that builds API docs](https://readme.com/customers) on ReadMe. <br>
+:monocle_face:   People that use API docs built on ReadMe. <br>
+:floppy_disk:   Folks that use any of our open source libraries.[^come-work-with-us] <br>
+:dancers:   [The ReadMe team](https://readme.com/about). <br>
```

Comment:
It might make more logistical sense for the footnote to be on this line instead of the one above.

Reply:
Oooh I hadn't even caught that! Would it be weird to have the footnote on both?

Reply:
![](https://i.kym-cdn.com/photos/images/original/000/538/731/0fc.gif)

Reply:
Probably a bit! I would keep it to just one, whichever makes more sense to you!

Reply:
cool cool, I'll move it down to the ReadMe team one!